ra_aid/__main__.py

```
@@ -206,6 +207,14 @@ def main():
 
             print_stage_header("Chat Mode")
 
+            # Get project info
+            try:
+                project_info = get_project_info(".", file_limit=2000)
+                formatted_project_info = format_project_info(project_info)
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)